15-Appendix.Rmd

```diff
@@ -0,0 +1,144 @@
+# (APPENDIX) Appendix {-}
+
+# Conda Package Management
+
+## Unified package management for both R and Python
+
+### Faster builds
+
+CRAN[^what_is_cran] mirrors do not provide linux[^linux_note] compiled binaries
+for packages. This means a long wait when doing `install.packages` both in an R
+Studio session and when running a Docker build for a `shiny` application.
+
+### Cross Language
+
+Both `dbtools` and `s3tools` rely on Python packages through the `reticulate` R
+to Python bridge. This means that `packrat` is not enough to reproducibly and
+reliably manage all of your application's dependencies.
+
+`conda` supports managing both Python and R dependencies in a single
+`environment`. It can make sure all of these libraries are compatible with each
+other.
+
+## Installing Packages
+
+If you need to find a package name you can use the [anaconda search] to find the
```

Comment:
Couldn't they just use `$ conda search PACKAGENAME`? 

[conda cheatsheet](https://conda.io/projects/conda/en/latest/user-guide/cheatsheet.html)

Reply:
I'll link it but I don't want them thinking they can have multiple environments